### ISSUE TITLE
toggleMaximizedPanel not working for side pannel

### ISSUE THREAD

**Issue by xvjiarui** [microsoft/vscode#141569]
Issue Type: <b>Bug</b>

The command `workbench.action.toggleMaximizedPanel` will toggle the panel instead of the side panel. 

The expected behavior should be toggle the side panel if side panel is in use. 

VS Code version: Code - Insiders 1.64.0-insider (45060913204279d4ecfe0575ca69d68211fb3798, 2022-01-26T05:26:19.708Z)
OS version: Darwin arm64 21.2.0
Restricted Mode: No
Remote OS version: Linux x64 5.4.0-96-generic
Remote OS version: Linux x64 5.4.0-96-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|8.00GB (0.14GB free)|
|Process Argv|--crash-reporter-id d9f6c2c8-fd52-4c94-a957-4a6395335605|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: batiquitos|
|OS|Linux x64 5.4.0-96-generic|
|CPUs|AMD EPYC 7542 32-Core Processor (128 x 1483)|
|Memory (System)|503.75GB (1.85GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: batiquitos|
|OS|Linux x64 5.4.0-96-generic|
|CPUs|AMD EPYC 7542 32-Core Processor (128 x 1483)|
|Memory (System)|503.75GB (1.85GB free)|
|VM|0%|
</details><details><summary>Extensions (29)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
vsc-material-theme|Equ|33.2.2
vsc-material-theme-icons|equ|2.2.1
better-cpp-syntax|jef|1.15.10
jupyter-keymap|ms-|1.0.0
remote-containers|ms-|0.215.0
remote-ssh|ms-|0.70.0
remote-ssh-edit|ms-|0.70.0
remote-wsl|ms-|0.64.2
latex-support|tor|3.10.0
vim|vsc|1.21.10
material-theme|zhu|3.13.16
path-intellisense|chr|2.7.0
gitlens|eam|11.7.0
vscode-diff|fab|1.4.2
seito-openfile|Fr4|1.8.8
copilot|Git|1.7.4688
vscode-pull-request-github|Git|0.35.2022012609
intellij-idea-keybindings|k--|1.5.0
MagicPython|mag|1.1.0
python|ms-|2021.12.1559732655
vscode-pylance|ms-|2022.1.3
jupyter|ms-|2022.1.1001614873
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6
vscode-yaml|red|1.3.0
code-spell-checker|str|2.1.4
python|tht|0.2.3
vscodeintellicode|Vis|1.2.16

(5 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vsdfh931:30280409
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
pythonvspyt200:30323110
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
wslgetstarted:30421357
vscop804:30404766
vscop453:30404998
vsclayoutctrt:30405800
azactmsal:30409287
pythonvspyt640:30426300
vsbas813:30426126

```

</details>

**Comment by xvjiarui** [comment]
Before 1.64.0, the behavior is toggle the side panel if side panel is in use.
So I guess there is a BC breaking in 1.64.0?

**Comment by taylorthurlow** [comment]
Yeah I think this is a pretty big oversight - as far as I can tell, I have zero ability to maximize anything located in the new side panel.

**Comment by adam-arold** [comment]
@sbatten this is **A BREAKING CHANGE**! You're breaking the worklfow for all of those people who **RELY ON THIS FUNCTIONALITY**. This is not something that you can put in the backlog. It is a **BREAKING CHANGE**!!!!

**Comment by adam-arold** [comment]
> Before 1.64.0, the behavior is toggle the side panel if side panel is in use. So I guess there is a BC breaking in 1.64.0?

It is not a guess, it is a **FACT**.

**Comment by adam-arold** [comment]
> @xvjiarui with the side panel functioning somewhat like a side bar and the panel still remaining at the bottom, there needs to be a new command for toggle maximized side panel.

NO, you need to **roll back** this change and re-release it in a way that it doesn't **break existing functionality**!

**Comment by adam-arold** [comment]
@bpasero @jrieken @joaomoreno @mjbvz @alexdima @sandy081 Please take a look at this, as it is definitely not a *feature request* and it is not something that should be voted on to be considered in some future release. This is a regression that was introduced with `1.64.0`.

**Comment by adam-arold** [comment]
> The command workbench.action.toggleMaximizedPanel will toggle the panel instead of the side panel.
>
> The expected behavior should be toggle the side panel if side panel is in use.

@xvjiarui I think that maximizing the *Panel* is still a valid use case for those who want that. As I outlined in #142120 which was closed the problem is that because the *Side Panel* was introduced it is not possible to move the *Panel* to the right. If I try to run that command I'll get

> Moving the panel with this command has been deprecated in favor of the "Move Views From Panel To Side Panel" and "Move Views From Side Panel To Panel" commands.

Moreover, the *Side Panel* is **different** from the *Panel*. It has different aesthetics and different functionality. The absolute minimum that should be done in order to fix this *breaking change* is to add a new command: `workbench.action.toggleMaximizedSidePanel`.

This still wouldn't solve the problem for those who were relying on the shortcut that you could use to move the *Panel* to left / bottom / right. This is from my downgraded `1.63.0` VS Code:

![image](https://user-images.githubusercontent.com/1253248/152516459-f5957482-0211-4bd6-be5b-53fe5d004319.png)

The right way to fix this issue is to let users *choose* whether they want the old *Panel* to work in the same way or to use the new *Side Panel*. This should be an option instead of forcing it on us and breaking existing workflows.

**Comment by carlfriedrich** [comment]
FYI: I [proposed a general flexible solution](https://github.com/microsoft/vscode/issues/141349#issuecomment-1029952188) to this in #141349.

**Comment by adam-arold** [comment]
> 🙂 This feature request received a sufficient number of community upvotes and we moved it to our backlog. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).
> 
> Happy Coding!

I wonder if it is possible to get some **human** attention. Preferably someone who *cares* about the end users.

**Comment by butuzov** [comment]
I rolledback to 1.63.0. honestly no idea what to do next. my ideal flow is broken =(

**Comment by wraiford** [comment]
For anyone else in my predicament just looking for the command name in default keybindings, the command to toggle the new Side Panel is `workbench.action.toggleAuxiliaryBar`.

**Comment by conradob** [comment]
@sbatten @wraiford `workbench.action.toggleAuxiliaryBar` does not have the same expected behaviour as `workbench.action.toggleMaximizedPanel`. We are still unable to *maximize* the auxiliary bar as we have with the panel.

A use case is to have the terminal on the auxiliary/side panel (on the right) and the remaining views on the standard panel (at the bottom)

There is no way to toggle maximize the auxiliary side panel.

**Comment by behzad-a** [comment]
> Due to the change in name/functionality of the panel and secondary side bar, this item is no longer relevant.

@sbatten does this mean there will not be support for maximizing the auxilliary bar at all? As @conradob mentioned, right now we can only toggle it, whereas before we could both toggle and maximize it. This seems like a regressions and breakage of user flows.

**Comment by sbatten** [comment]
There is no regression, this is a new feature request. You can still move the panel to the right/left and maximize as you could before the Secondary Side Bar existed. A new feature request should be filed for the Secondary Side Bar as this issue was created when the Side Panel was intended to replace panel movement. We have since changed courses on that.
